lib/util/content.py

```diff
@@ -34,24 +34,38 @@ def get_datastream():
     return datastream
 
 
+def _find_playbooks():
+    if user_content:
+        build_content(user_content)
+        return user_content / 'build' / 'ansible'
+    else:
+        return Path('/usr/share/scap-security-guide/ansible')
+
+
 def get_playbook(profile):
     if user_content:
         build_content(user_content)
         playbook = user_content / 'build' / 'ansible' / f'rhel{rhel.major}-playbook-{profile}.yml'
     else:
```

Comment:
Why did you leave this in the function? ... Isn't it already done by `_find_playbooks()` ?